src/org/usfirst/frc2175/subsystem/RobotSubsystems.java

```diff
@@ -22,6 +24,7 @@ public RobotSubsystems(RobotConfig robotConfig,
         this.driverStation = driverStation;
 
         dreamIntakeSubsystem = new DreamIntakeSubsystem(robotConfig);
```

Comment:
We can remove `dreamIntakeSubsystem` from `RobotSubsystems` by the sound of things. Don't delete any dream-related files though.